reconcile/utils/terraform_client.py

```diff
@@ -414,6 +416,11 @@ def construct_oc_resource(self, name, data, account):
             "data": {}
         }
 
+        anno_dict = json.loads(base64.b64decode(annotations))
+
+        for k, v in anno_dict.items():
+            body["metadata"]["annotations"][k] = v
```

Comment:
Easier:
```suggestion
        body["metadata"]["annotations"][k] = anno_dict
```

Reply:
Perhaps this suggestion instead? Otherwise we would overwrite whatever existed in `annotations`.
```suggestion
        body["metadata"]["annotations"].update(anno_dict)
```

Reply:
Even better. :)